security/package-requirements.md

```diff
@@ -0,0 +1,16 @@
+## SSWG security requirements for packages on the index
+
+This information is intended for package authors who have packages listed on the SSWG's package index or are looking into pitching/proposing their package to be listed. If instead you found (or have heard of) a security vulnerability you’d like to report, please have a look over [here](contributor-found-vulnerability.md).
+
+---
+
+Packages that are listed on the [SSWG's package index](https://swift.org/server/#projects) are required to follow the following guidelines around security.
+
+Where security vulnerabilities are involved, it is key to ensure that somebody who discovers a vulnerability in your package can immediately find information on how to report it. As the package author, you know best where to put important information about your software. In general, it is typically the `README.md` file in the root of your repository on GitHub/GitLab/etc. The phrase "main information page" is used elsewhere in this document to refer to the `README.md` or other resource as appropriate for your project.
+
+The key requirements are:
```

Comment:
Should the "Project authors are also encouraged to make use of their source control system security features" section from `package-maintainer-received-vulnerability-report.md` possibly appear here as well?

Reply:
good idea, done!